cargo-apk/CHANGELOG.md

```diff
@@ -1,5 +1,7 @@
 # Unreleased
 
+- Fix NativeActivity library path on non-UNIX systems, which resulted in runtime crashes on startup.
```

Comment:
This only applies to `cargo-apk`, right? I don't think we need to perform any minor/major dependency bump here?

Reply:
Patch releases for `cargo-apk` and `ndk-build` would be good and `^` enforcing the `ndk-build` patch

Reply:
Feel free to do that :) - maybe mention just the version upgrade here and refer to the `ndk-build` changelog or adopt the line below?

Btw caret `^` semver matching is the default, meaning that least-significant (patch for 0.x.y, or minor for x.y.z) versions may always be bumped. But it's good to turn the patch into a `.1` to clarify that at least that version has to be used.